### PR TITLE
controllers: configure log rotater spec in csi-op operatorconfig

### PR DESCRIPTION
**Pull request by leelavg**
log rotation feature in csi-op provides a way to rotate csi logs based on custom configuration and this PR configures the default agreed upon values.

### CODE REVIEW

File: pkg/templates/csi.go [hunk 1]
```diff
@@ -73,17 +74,42 @@ const CSIOperatorConfigName = "ceph-csi-operator-config"
 
 var CSIOperatorConfigSpec = csiopv1a1.OperatorConfigSpec{
 	DriverSpecDefaults: &csiopv1a1.DriverSpec{
+		Log: &csiopv1a1.LogSpec{
+			Rotation: &csiopv1a1.LogRotationSpec{
+				Periodicity: csiopv1a1.DailyPeriod,
+				MaxLogSize:  resource.MustParse("500M"),
+				MaxFiles:    7,
```
Comment (parth-gr) [inline pkg/templates/csi.go:81]: do we define this as a constants values?

**Review by parth-gr — COMMENTED** [review]
lgtm

Try running the changes on openshift cluster, and validate that things are working fine, plus see the pod yaml and validate that csi pod actually has the logrotate container added

File: pkg/templates/csi.go (hunk 1 above)
Comment (leelavg) [reply pkg/templates/csi.go:81]: I have no preference of one over other however I have my reservations on when to use const, mostly for invariants across many releases, env var, cross-pkg knowledge etc and I don't deem above should be a const.

**Comment by nb-ohad** [comment]
LGTM
@leelavg If you have no other reservation please merge

**Comment by leelavg** [comment]
I'm still not able to test it, cluster availability, some other work etc. Will try to close it before EOD.

**Comment by leelavg** [comment]
> Try running the changes on openshift cluster, and validate that things are working fine, plus see the pod yaml and validate that csi pod actually has the logrotate container added
- verified.

**Pull request merged**